docs/parcels.md

```diff
@@ -0,0 +1,109 @@
+# Parcels
```

Comment:
Can you add more about the benefits and possibly any drawbacks more towards the top of this document?  As a reader you want to know quickly how this applies to you - "why do I care about parcels?" - without having to read too far into the article.

Also, can you describe a relationship between applications and parcels?  specifically can an app contain an app contain an app?  can a parcel contain a parcel?  can a parcel contain an app?

Consistent capitalization of Application and Parcel

Reply:
Hmmm... let's talk about this.

On why: I don't anticipate this page being the page that sells parcels this is the page that _explains_ parcels. I'm assuming you already care when you start reading.

On the relationships: Applications are microapps that are declaratively mounted/unmounted by single-spa. Parcels are a bundle of functionality (small or large) that are imperatively mounted/unmounted by the developer. So using those definitions the only thing that can mount an Application is single-spa, but you could potentially not register an application and instead mount the application as a parcel. So Apps can mount parcels, and parcels can mount parcels, but an app can't mount an app.